_includes/default-header.html

```diff
@@ -42,6 +41,7 @@
               <a class="navbar-item" href="/ols-6#mentors"> Mentors & Experts </a>
               <a class="navbar-item" href="/ols-6#collaborators"> Collaborators </a>
               <a class="navbar-item" href="/ols-6#resources"> Resources </a>
+              <a class="navbar-item" href="/cohorts"> Previous Cohorts </a>
```

Comment:
I do not think inside OLS-6 dropdown menu is the best location. I would prefer to leave in the OLS program drop-down menu